### ISSUE TITLE
"evaluation error: not a float expression" passing float array to function

### ISSUE THREAD

**Issue by sbronson** [MiniZinc/libminizinc#628]
```
function bool: f(array [1..1] of 1.0..1.0: a) = true;
constraint f([1.0]);
```
```
$ minizinc -c f.mzn
Error: evaluation error: not a float expression
f.mzn:2.12-19
  in call 'f'
  in array literal
```
On develop.

**Comment by Dekker1** [comment]
Looks like this is an issue in the checking of the domain of the array that is being passed in to the `f` function. The issue was already resolved for integers, but the floating point version must have been forgotten.
I've submitted an patch to our CI that will hopefully be merged shortly.

**Comment by cyderize** [comment]
This should now be fixed in version 2.7.0
